src/site/content/en/fast/minify-css/index.md

```diff
@@ -67,8 +67,8 @@ The resulting report shows that up to **16KB** can be saved from the **animate.c
 Now you'll inspect the content of the CSS:
 
 1. Open the [Fav Kitties site](https://fav-kitties-animated.glitch.me/) in Chrome (it might take a while for Glitch servers to respond the first time).
```

Comment:
```suggestion
1. Open the [Fav Kitties site](https://fav-kitties-animated.glitch.me/) in Chrome. (It might take a while for Glitch servers to respond the first time.)
```